mrtt-ui/src/data/questions.js

```diff
@@ -0,0 +1,63 @@
+export default {
+  ecologicalAims: {
```

Comment:
Im thinking its easier not to break into sub sections. Then we can just look up questions programatically without caring what section they come from.

Reply:
I believe it is an important differentiator - I want to avoid running into issues where sections may have similar or the same names. Also just makes the code more easy to comprehend.

Reply:
Sure. Consistent too vs questionMapping.